var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -245,6 +249,9 @@ def configure_args(self):
         else:
             config_args.append("--disable-pnetcdf")
 
+        if "~byterange" in self.spec:
+            config_args.append("--disable-byterange")
```

Comment:
I'm pretty sure byte-range is already off by default, at least in 4.7.0-4.9.0, so it might be good to either invert the logic or maybe use `self.enable_or_disable` (though I don't know how that works with variants that don't exist for a given version).

Reply:
Ok, I'll need to test that logic on my Mac before I push the changes.

Reply:
Turns out the `self.enable_or_disable` is already in the file, just a few lines up, so I will remove 252 and 253.